dotcom-rendering/src/palette.ts

```diff
@@ -5194,6 +5198,10 @@ const paletteColours = {
 		light: discussionPaginationBorderLight,
 		dark: discussionPaginationBorderDark,
 	},
+	'--discussion-grey': {
```

Comment:
What about `--discussion-subdued` or `--discussion-faded`? Would that be more semantic about what that colour represents?